### PR TITLE
Additional changes to PDF mode interface

### PR DESCRIPTION
**Pull request by mttwc**
- Reorder options to be before the section picker
- Popover as component when the selected page range is bad
- Increased perf (fail-fast) when parsing page range
- Fade in animations for More options
- Styling to redlines

### CODE REVIEW

**Comment by mttwc** [comment]
TODO: Third party notices file

File: src/scripts/clipperUI/components/pdfClipOptions.tsx [hunk 1]
```diff
@@ -91,46 +95,65 @@ class PdfClipOptionsClass extends ComponentBase<PdfClipOptionsState, ClipperStat
...
-				{!pdfPreviewInfo.allPages ?
-					<input type="text" id={Constants.Ids.rangeInput} class={invalidClassName} placeholder="e.g. 1-5, 7, 9-12" value={this.props.clipperState.pdfPreviewInfo.selectedPageRange} {...this.enableInvoke(this.onSelectionChange, 62, false) }></input>
-					: <span class="pdf-label">{Localization.getLocalizedString("WebClipper.Preview.Header.PdfPageRangeRadioButtonLabel")}</span>}
-				{!pdfPreviewInfo.allPages && pageRangeIsInvalid ?
-					<div class="popover">{Localization.getLocalizedString("WebClipper.Preview.Header.PdfInvalidPageRange")}</div>
-					: ""}
+				<input
+					type="text"
+					id={Constants.Ids.rangeInput}
+					class={invalidClassName}
```
Comment (VishaalK) [inline src/scripts/clipperUI/components/pdfClipOptions.tsx]: Gonna change this to `className`

File: src/scripts/clipperUI/components/popover.tsx [hunk 2]
```diff
@@ -0,0 +1,70 @@
+import {Constants} from "../../constants";
+
+import {ComponentBase} from "../componentBase";
+
+import { StringUtils } from "../../stringUtils";
```
Comment (VishaalK) [inline src/scripts/clipperUI/components/popover.tsx]: Gonna fix this spacing

**Comment by VishaalK** [comment]
Changed instances of class to className, re-ordered the PDF button and fixed tests, and some styling issues

I think it is good to go now!!

**Pull request merged**